.github/workflows/test-julia.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.5'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+      - uses: actions/cache@v1
+        env:
+          cache-name: cache-artifacts
```

Comment:
Do we need both `cache-name` and the `-test-` component? I'm thinking these could be combined/shortened. Maybe something like `${{ runner.os }}-artifacts-${{ hashFiles('**/Project.toml') }}`.

Reply:
I have no idea. I'll change it per your suggestion.

Reply:
Done.